osm2pgsql-el7/Dockerfile

```diff
@@ -0,0 +1,52 @@
+# osm2pgsql
+# Using this because it has Ruby pre-installed 
+# as we were having trouble installing Ruby version >= 2.4.0 
+# (which is required for the fpm gem).
+FROM drecom/centos-ruby
```

Comment:
Need to tag all these `FROM` lines with the exact ruby version.

Reply:
Also are we not using an older fpm? You're specifying fpm 1.6.3 below, isn't that quite old?